Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:buster as base
+
+ENV PYTHONPATH=${PYTHONPATH}:${PWD} 
+RUN pip3 install poetry gunicorn markupsafe==2.0.1
+RUN mkdir /todo_app 
```

Comment:
Not important, but worth noting that you don't need this line - COPY will happily create the directory for you